chapters/en/_toctree.yml

```
@@ -39,10 +82,80 @@
   - local: Unit 4 - Multimodal Models/supplementary-material
     title: Supplemental reading and resources
  
-- title: Unit 5. Multimodal Models
+- title: Unit 5. Generative Models
+  - sub-title: Diffusion models
+    sections:
+    - local: Unit 5 - Generative Models/Diffusion models/Introduction - Diffusions
+      title: Introduction to Diffusion Models
+    - local: Unit 5 - Generative Models/Diffusion models/Simple explanation and showcase of external resources
+      title: Extras
+    - local: Unit 5 - Generative Models/Diffusion models/stable_diffusion
+      title: Introduction to Stable Diffusion
+  - sub-title: GANs & VAEs
+    sections:
+    - local: Unit 5 - Generative Models/GANs & VAEs/StyleGAN
+      title: StyleGAN Variants
+  - sub-title: Introduction
+    sections:
+    - local: Unit 5 - Generative Models/Introduction/Introduction
+      title: Introduction
+  - sub-title: PRACTICAL APPLICATIONS & CHALLENGES
+    sections:
+    - local: Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues
```

```suggestion
    - local: Unit 5 - Generative Models/Practical Applications & Challenges/Ethical Issues
```
Can we change the folder name for consistency?